lib/src/Base/Stat/openturns/Sample.hxx

```diff
@@ -340,6 +341,10 @@ public:
   /** Get the marginal sample corresponding to indices dimensions */
   Sample getMarginal(const Indices & indices) const;
 
+  /** Select points in the sample */
+  Sample select(const UnsignedIntegerCollection & indices) const;
+  Sample select(const Indices & indices) const;
+
```

Comment:
select should be documented

Reply:
Of course ;-) *The documentation will follow once we agreed on the code.* as I said!

Reply:
Oh, sorry

Reply:
In fact there is no need to define 2 methods, since Indices inherits from UnsignedInterCollection.

Reply:
@regislebrun Remove select(Indices), it will automatically speed up SWIG version of Sample.select.

Reply:
@dbarbier I first defined the Indices version because Indices has became the usual way of dealing with collections of integers, but Collection<UnsignedInteger> is not automatically converted into Indices. So I added the second version. But I was wrong: it was the other way to go! Thanks.

Reply:
@dbarbier Removing the useless Sample::select(Indices) makes the sample selection option of my benchmark 10% faster than the new BootstrapExperiment::generate(), which is already 30% faster than the pure np version! Thanks.

Reply:
The mix between np and OT is 26x slower than the best 100% OT version and 17x slower than the 100 np version. Users should avoid mixing the two packages when possible, the conversion between samples ans np arrays is awfully slow.

Reply:
@regislebrun I do not understand your timings, can you please show your output?

Reply:
Ok, got it

Reply:
@regislebrun See my branch db/swig-buffer; I have to test and polish it.